src/components/LibrariesList.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from "react";
+import { PanelGroup } from "react-bootstrap";
+import { Store, Reducer } from "redux";
+import { State } from "../reducers/index";
+import { LibrariesData, LibraryData } from "../interfaces";
+import { connect } from "react-redux";
+import ActionCreator from "../actions";
+import LibrariesListItem from "./LibrariesListItem";
+
+export interface LibrariesListStateProps {
+  libraries?: LibrariesData;
+}
+
+export interface LibrariesListOwnProps {
+  store?: Store<State>;
+}
+
+export interface LibrariesListDispatchProps {
+  fetchData: () => Promise<LibrariesData>;
+}
+
+export interface LibrariesListProps extends LibrariesListStateProps, LibrariesListOwnProps, LibrariesListDispatchProps {};
+
+export class LibrariesList extends React.Component<LibrariesListProps, void> {
+
+  render(): JSX.Element {
+    let hasLibraries = (this.props.libraries && this.props.libraries.libraries && this.props.libraries.libraries.length);
+    return(
+      <ul className="list panel-group">
+        { hasLibraries ?
+          this.props.libraries.libraries.map(library =>
+            <LibrariesListItem
+              key={library.uuid}
+              library={library}
+              store={this.props.store}
+            />
+          ) :
+          <h1 className="page-header">There are no libraries in this registry yet.</h1>
```

Comment:
The welcome message, or rather, the name of the app, should be an h1. This is more like an h2 or maybe not even a heading at all since it's not defining something on the page.